### ISSUE TITLE
withdrawal: Support selecting custom sat per byte

### ISSUE THREAD

**Issue by tiero** [tdex-network/tdex-app#447]

**Comment by Janaka-Steph** [comment]
Should it be a simple input, a slider?

**Comment by librepgp** [comment]
Instead of slider I would like to use large buttons.

- This is a similar approach to BlueWallet & GreenWallet.
- People are already used to this UX pattern in GW (the most used liquid wallet).

![ux-proposal](https://user-images.githubusercontent.com/45250003/153118118-93c6b4ff-105c-4fca-8265-5834eec26831.png)

Please let me know if you would like a different direction @Janaka-Steph

**Comment by tiero** [comment]
Agreed. We can re-use the same modal component used in Settings > L-BTC Unit (without the search box)

**Comment by librepgp** [comment]
Perfect.

I've been getting familiarized with the codebase and setting up the environment these past two days. I'll submit a PR tomorrow morning to track the progress.

In the meantime I'll send you guys some screenshots of the UI I'm working on.

<img width="500" alt="Screen Shot 2022-02-10 at 14 23 29" src="https://user-images.githubusercontent.com/45250003/153507483-3a042acf-0e1b-4ee6-9862-3db12e20bcda.png">

I like this approach because:
- It is non-invasive. 
- It has a sane default. 
- It allows customization only if the user wants to.

**Comment by librepgp** [comment]
Thanks for the feedback.

I'll do the custom fee like the second screenshot in BW. 

At the bottom where it says "Personalizada" it means "Custom amount". You click it and then an input dialog pops up.

**Comment by librepgp** [comment]
I'm done with the second screen. Any feedback on this new screen would be great :D @Janaka-Steph 

I'll make sure to include the changes you suggested in the first screen. 

The "NETWORK FREE" label and the "SAT/VBYTE" button I added are actually aligned with the "MAX"/"FIAT VALUE" labels above. I'll change the alignment and test out the other changes as well.

<img width="395" alt="screen2" src="https://user-images.githubusercontent.com/45250003/153684276-954b0171-2bc1-4ba0-9536-e2e378cfd30e.png">


<img width="345" alt="screen1" src="https://user-images.githubusercontent.com/45250003/153684271-4223e52a-cab7-4f35-94b2-2b7025df3271.png">

**Comment by Simo29** [comment]
please align with the design team for all the changes that impact look& fell/ visual
